include/openthread/platform/radio.h

```diff
@@ -92,21 +92,33 @@ typedef enum otRadioCaps {
     OT_RADIO_CAPS_CSMA_BACKOFF     = 8, ///< Radio supports CSMA backoff for frame transmission (but no retry).
 } otRadioCaps;
 
+/**
+ * This structure represents the Header IEs related information of a radio frame.
```

Comment:
suggestion: This may be first place someone looking at code will see the term  "Header IE". How about defining the abbreviation here as well?